apps/theming/lib/ImageManager.php

```diff
@@ -223,7 +223,7 @@ public function updateImage(string $key, string $tmpFile) {
 			throw new \Exception('Unsupported image type');
 		}
 
-		if ($key === 'background' && strpos($detectedMimeType, 'image/svg') === false) {
+		if ($key === 'background' && strpos($detectedMimeType, 'image/svg') || strpos($detectedMimeType, 'image/gif') === false) {
```

Comment:
```suggestion
		if ($key === 'background' && (strpos($detectedMimeType, 'image/svg') === false || strpos($detectedMimeType, 'image/gif') === false)) {
```

Otherwise this will not trigger on svg images anymore.